### DISCUSSION TITLE
Would be nice to add command line support

### DISCUSSION THREAD

**Discussion by lobeg25** [zodiacon/ObjectExplorer#2] [Ideas]
I'm really grateful for the time you took to make this tool. Zombie processes are not easy to identify on Windows. The only I had (apart from a tool that worked in the time of Windows 8.1) was sysinternals RamMap, with which you can spot zombie processes as listed processes using 0 K of private memory.

It would be great to be able to call ObjectExplorer on the command line. I would in particular be interested in a way to list zombie processes (and, why not, zombie threads).

**Comment by zodiacon** [comment]
Adding CLI is a good idea, adding to my list :)

**Reply by NiklasEMS** [reply]
Yes, it would be great with CLI so you can schedule it (scheduled task) to produce lists to compare and see growth over time. Also good to just ask others to run e g : ObjExp.exe -ZombieProcesses -ZombieThreads -out FileList.txt

Perhaps even include such in tool also where you start it and it will look for zombie processes every x seconds/minutes and list the longest living orphaned processes.
